Controllers/note.js

```diff
@@ -0,0 +1,91 @@
+const _ = require('lodash');
+const Note = require('../Models/note');
+
+const getAllNotesOfAUser = async (req, res) => {
+  const { userId } = req.params;
+  try {
+    const response = await Note.find({ userId });
+    res.json({ response });
+  } catch (error) {
+    res.status(400).json({ response: error.message });
```

Comment:
consistency with API responses can be followed.

in category.js the response is different and in this file the response format is different. (status code missing in category.js APIs as well)

Reply:
This has been taken care of